### PR TITLE
Unit test ticket quality computation

### PR DESCRIPTION
**Pull request by masih**
Test ticket quality
* is weighed by power
* is as expected for edge case power value
* handles unexpected ticket length gracefully

Part of #9

### CODE REVIEW

File: gpbft/ticket_quality_test.go [hunk 1]
```diff
@@ -61,3 +64,56 @@ func FuzzTQ_linearToExp(f *testing.F) {
...
+	t.Run("Zero power is handled gracefully", func(t *testing.T) {
+		ticket := generateTicket(t)
+		quality := ComputeTicketQuality(ticket, 0)
+		require.True(t, math.IsInf(quality, 1), "Expected quality to be infinity with power 0, got %f", quality)
+	})
+
+	t.Run("Negative power is handled gracefully", func(t *testing.T) {
+		ticket := generateTicket(t)
+		quality := ComputeTicketQuality(ticket, -5)
+		require.Negative(t, quality, "Expected negative quality for negative power, got %f", quality)
```
Comment (Stebalien) [inline gpbft/ticket_quality_test.go:92]: We want this to be infinity.

**Review by Stebalien — APPROVED** [review]
